intro-Linux/ch_uploading_files.tex

```diff
@@ -2,25 +2,29 @@ \chapter{Uploading/downloading/editing files}
 
 \subsection{Uploading/downloading files}
 
-Detailed information about uploading/downloading files with ``scp'' (using an OS
-X or Linux client) or WinSCP (Windows client), see
-\url{http://hpc.ugent.be/userwiki/index.php/User:VscCopy}.
+To transfer files from and to the HPC, see
+\href{\HPCManualURL#sec:filetranfer}{the section about tranferring files in the HPC manual}.
+
+\ifwindows
 
 After copying files it is advised to run
 
 \begin{prompt}
 %\shellcmd{dos2unix filename}
 \end{prompt}
 
-as this will fix any problems with windows / unix conversion.
+as this will fix any problems with Windows/Unix conversion.
```

Comment:
Is it worth mentioning here what kind of errors you can run into?

Not sure how consistent those errors are though...

Reply:
I think you might run into different errors depending on the kind of file you transfer.

Reply:
Maybe try and see what happens if you provide a job script with Windows line endings to `qsub`, that's something we see frequently in HPC-UGent intro sessions.

Reply:
Done. Btw, I just ran my very first job on the HPC!